homeassistant/components/generic_hygrostat/config_flow.py

```diff
@@ -0,0 +1,129 @@
+"""Config flow for Generic Hygrostat integration."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any
+
+import voluptuous as vol
+from voluptuous.schema_builder import UNDEFINED
+
+from homeassistant import config_entries
+from homeassistant.components.humidifier import HumidifierDeviceClass
+from homeassistant.const import CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import config_validation as cv, selector
+
+from .const import (
+    CONF_AWAY_FIXED,
+    CONF_AWAY_HUMIDITY,
+    CONF_DEVICE_CLASS,
+    CONF_DRY_TOLERANCE,
+    CONF_HUMIDIFIER,
+    CONF_INITIAL_STATE,
+    CONF_KEEP_ALIVE,
+    CONF_MAX_HUMIDITY,
+    CONF_MIN_DUR,
+    CONF_MIN_HUMIDITY,
+    CONF_SENSOR,
+    CONF_STALE_DURATION,
+    CONF_TARGET_HUMIDITY,
+    CONF_WET_TOLERANCE,
+    DEFAULT_MAX_HUMIDITY,
+    DEFAULT_MIN_DUR,
+    DEFAULT_MIN_HUMIDITY,
+    DEFAULT_NAME,
+    DEFAULT_TOLERANCE,
+    DOMAIN,
+)
+
+
+def generate_schema(existing_data: Mapping[str, str]):
```

Comment:
Please make the schema a constant, and use `add_suggested_values` helper instead.